### DISCUSSION TITLE
Autocomplete in VSCode

### DISCUSSION THREAD

**Discussion by Mark2Mark** [coldtype/coldtype#77] [Q&A] [answered]
I wonder how you made VSCode to do autocomplete for Coldtype as seen in your YT video?

Seems Handy due to these one- or two-character method names.

**Comment by stenson** [comment, marked as answer]
Definitely! Recently made a short video about this, mean to post it here actually — basically just boils down to selecting the venv python binary into the lower bar of the VS Code window: https://www.youtube.com/watch?v=B_jtHWeYRqY

You might also need to run the Command Palette (⌘⇧P) and then run `Developer → Reload Window` to get everything to refresh properly. Let me know if that doesn't do the trick!

**Comment by Mark2Mark** [comment]
Oh yeah! This rocks! Thanks so much. 
I had a messed workspace before, which did not recognise the venv python. Now it did.

My steps are:

- Open the Coldtype directory (with the venv inside) in VSCode
- <kbd>⌘ ⇧ P</kbd> `Python: Create Terminal`*
- *then* `source venv/bin/activate`
- Only then the venv python interpreter shows up in the list

*) Before I just opened the terminal via `New Terminal`, which worked for the animations, but didn’t recognise the venv py interpreter.

**Reply by stenson** [reply]
Ah nice — very helpful to keep here as a reference for future readers!
